yarn.lock

```diff
@@ -3575,50 +3568,6 @@ defaults@^1.0.3:
   dependencies:
     clone "^1.0.2"
 
-defender-admin-client@1.11.0:
-  version "1.11.0"
-  resolved "https://registry.yarnpkg.com/defender-admin-client/-/defender-admin-client-1.11.0.tgz#9d2684c597944e22c1d5a9136f71e68c832fa0c5"
-  integrity sha512-ojKNkYvpBs9NPTxQC5F/KbkZ8BIgQv3px20Ev268gLupx2cQqWgl2SOlmo/sacvO3ffXCaAZWgkR8ZtTp+j++w==
-  dependencies:
-    axios "^0.19.2"
-    defender-base-client "1.10.0"
-    lodash "^4.17.19"
-    node-fetch "^2.6.0"
-
-defender-autotask-client@1.10.0:
-  version "1.10.0"
-  resolved "https://registry.yarnpkg.com/defender-autotask-client/-/defender-autotask-client-1.10.0.tgz#1945e88603a4619a0f1cf9f182cadc34de4cad7c"
-  integrity sha512-RuyBUEDuCqeyWj594VLY/wW0J+5MFoCRny39Kzx0oxbTRVZWKUeyfXs9ats4COcYVYha90lYktnrgZDcSqo74Q==
-  dependencies:
-    axios "^0.19.2"
-    defender-base-client "1.10.0"
-    dotenv "^10.0.0"
-    glob "^7.1.6"
-    jszip "^3.5.0"
-    lodash "^4.17.19"
-    node-fetch "^2.6.0"
-
-defender-base-client@1.10.0:
-  version "1.10.0"
-  resolved "https://registry.yarnpkg.com/defender-base-client/-/defender-base-client-1.10.0.tgz#d85af9e84f57d6201e7741e44e59d5a3ff9c2b75"
-  integrity sha512-PvjJj4pWug/PVZWVi6jrdXGpykgtifGdomV1ePgZSj7cjK2mSbS0P3GUUeIzCA/d5HSV5/L6reZkI1KavnSdpA==
-  dependencies:
-    amazon-cognito-identity-js "^4.3.3"
-    axios "^0.19.2"
-    lodash "^4.17.19"
-    node-fetch "^2.6.0"
-
-defender-relay-client@1.11.1:
-  version "1.11.1"
-  resolved "https://registry.yarnpkg.com/defender-relay-client/-/defender-relay-client-1.11.1.tgz#0ae73c1ca972f7fd082ab0c3a37301cf96a32b77"
-  integrity sha512-gz2CA45m5xho3p/nbKCVC/j07kbM/8hQajpEr+dl2YJbWRSFuxePkkd3TFKl0bya1MDZccHW8lbCQs47PQ+18w==
-  dependencies:
-    amazon-cognito-identity-js "^4.3.3"
-    axios "^0.19.2"
-    defender-base-client "1.10.0"
-    lodash "^4.17.19"
-    node-fetch "^2.6.0"
-
```

Comment:
This change surprises me, how come we are removing this dependencies?

Reply:
Mmm that's an interesting question.
I am not sure, I reverted the changes and ran `yarn` again and they got deleted again.

My feeling is that it has something to do with https://github.com/OpenZeppelin/defender-client/pull/110. Seems like nobody ran `yarn` since it was merged